packages/vscode-extension/src/handlers.ts

```diff
@@ -375,9 +374,18 @@ export async function createNewProjectHandler(args?: any[]): Promise<Result<any,
   const res = result.value as CreateProjectResult;
   const projectPathUri = Uri.file(res.projectPath);
   // show local debug button by default
+  const triggerFrom = getTriggerFromProperty(args);
   if (isValidOfficeAddInProject(projectPathUri.fsPath)) {
+    ExtTelemetry.sendTelemetryEvent(TelemetryEvent.CreateProjectStart, {
+      ...triggerFrom,
+      [TelemetryProperty.IsOfficeAddIn]: "true",
+    });
     await openOfficeDevFolder(projectPathUri, true, res.warnings, args);
   } else {
+    ExtTelemetry.sendTelemetryEvent(
```

Comment:
Better add [TelemetryProperty.IsOfficeAddIn]: "false" even when it's not Office Add-in.